app/services/workers/adjustments-worker.js

```diff
@@ -106,17 +107,14 @@ var processAdjustments = function (task) {
   })
 }
 
-var updateAdjustmentsStatus = function (task) {
+var updateAdjustmentsStatus = function (workloadStagingIdStart, workloadStagingIdEnd, workloadReportId) {
   var idsMap = new Map()
   idsMap.set(adjustmentStatus.ACTIVE, [])
   idsMap.set(adjustmentStatus.SCHEDULED, [])
   idsMap.set(adjustmentStatus.ARCHIVED, [])
 
-  var workloadIdStart = task.additionalData.workloadBatch.startingId
-  var workloadIdEnd = workloadIdStart + task.additionalData.workloadBatch.batchSize - 1
-
-  logger.info('Retrieving open adjustments for workload owners with workloads ' + workloadIdStart + ' - ' + workloadIdEnd)
-  return getAppAdjustmentsForBatch(workloadIdStart, workloadIdEnd)
+  logger.info('Retrieving open adjustments for workload owners with workloads\' staging ids ' + workloadStagingIdStart + ' - ' + workloadStagingIdEnd)
```

Comment:
Logging statement probably now needs to include the Workload Report Id in order to always be correct.